docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Nexus zkVM SDK Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+import { Steps } from "nextra/components";
+
+## Reproducible Proving and Verification
```

Comment:
The first heading in this page should match the title on line 2.

Reply:
Fixed in [36f79a4](https://github.com/nexus-xyz/nexus-zkvm/pull/271/commits/36f79a4ffc509fe44f4c288746c674652d698a4b)